chickadee/processes/wps_BCCAQ.py

```diff
@@ -22,7 +20,10 @@ class BCCAQ(Process):
     output to a fine spatial resolution"""
 
     def __init__(self):
-        self.status_percentage_steps = common_status_percentage
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{"get_ClimDown": 5, "parallelization": 10, "set_end_date": 15},
+        )
```

Comment:
Instead of casting here we could use `.update()` to put together a `dict`.

Reply:
@nikola-rados I've tried that method first but the problem is that `.update()` returns `NoneType` and only updates the original `common_status_percentages` . In order to use the method, there needs to be one more line:
```
common_status_percentages.update({"get_ClimDown": 5, "parallelization": 10, "set_end_date": 15})
self.status_percentage_steps = common_status_percentages
```
I don't think it is ideal to update the original dictionary, which might affect other processes.

Reply:
Agreed, we can stick with what you have then!